examples/e-commerce/src/theme.css

```diff
@@ -196,52 +200,41 @@ a[class^='ais-'] {
 .ais-SearchBox-reset,
 .ais-SearchBox-loadingIndicator,
 .ais-SearchBox-submit {
+  align-items: center;
   -webkit-appearance: none;
   -moz-appearance: none;
   appearance: none;
+  display: flex;
   height: 100%;
+  justify-content: center;
   position: absolute;
   top: 50%;
   transform: translateY(-50%);
+  width: 48px;
```

Comment:
This makes the buttons in the input more accessible on mobile: it increases the touch area.